core/src/main/java/org/jboss/jandex/WildcardType.java

```diff
@@ -26,7 +26,7 @@
  * @author Jason T. Greene
  */
 public class WildcardType extends Type {
-    private static Type OBJECT = new ClassType(DotName.OBJECT_NAME);
+    private static final Type OBJECT = new ClassType(DotName.OBJECT_NAME);
```

Comment:
OT: it might be useful to make this kind of constants public. WDYT? Other candidates are declared on the `org.jboss.jandex.DotName`, e.g. `DotName.OBJECT_NAME`.

Reply:
Making some of the `DotName`-typed constants on `DotName` public certainly makes sense. Definitely not this one, which is on `WildcardType` and is of type `ClassType` :-)

Reply:
For sure, it should go to the `ClassType`. Ha, there is `org.jboss.jandex.ClassType.OBJECT_TYPE`!

Reply:
Done.